pkg/resources/resource_source_postgres.go

```diff
@@ -34,7 +35,7 @@ var sourcePostgresSchema = map[string]*schema.Schema{
 		Optional:    true,
 	},
 	"table": {
-		Description: "Creates subsources for specific tables. If neither table or schema is specified, will default to ALL TABLES",
+		Description: "Creates subsources for specific tables in the Postgres connection.",
 		Type:        schema.TypeList,
```

Comment:
This will need to be converted to `TypeSet` to ignore order which cannot be guaranteed between how the user enters the `table` values and what the system catalog query will return.